### ISSUE TITLE
Conflict with ccc causing E565

### ISSUE THREAD

**Issue by Soliprem** [jmbuhr/otter.nvim#178]
When both otter and ccc are enabled, upon the activation of the injected lsp and its use, any modification of the text within the buffer triggers a message (image below). 
![image](https://github.com/user-attachments/assets/b156d6f8-047d-4466-baa2-0ae5329cd762)

A similar conflict occurred between coq_nvim and nvim-ts-autotag, and was solved with [this PR](https://github.com/windwp/nvim-ts-autotag/pull/101) by nvim-ts-autotag

**Comment by Soliprem** [comment]
hey @jmbuhr, when I tested the same interaction, I get the same error :/

**Comment by jmbuhr** [comment]
Perfect. Is that the complete text of error E565?

**Comment by jmbuhr** [comment]
also, what is this ccc and how do I reproduce this? Otter was meant to only throw a warning, but I didn't catch the correct error, yet.

**Comment by Soliprem** [comment]
yep, that I can see that's all of it. Here, there's also a screenshot with the warning I think you're talking about. The warning appears the first time I type after activating otter, while the error shoots every time.

![Image](https://github.com/user-attachments/assets/27b33cc2-2e35-4bd9-8ec4-05452a1113eb)

ccc is a [plugin](https://github.com/uga-rosa/ccc.nvim)

**Comment by jmbuhr** [comment]
Can you try with the `fix/ccc` branch (https://github.com/jmbuhr/otter.nvim/pull/219)? It should now only throw the warning if some other error occurs, not the textlock error.

**Comment by jmbuhr** [comment]
what nvim version are you on btw.?

**Comment by Soliprem** [comment]
version is 0.10.4
```🞈 : ./result/bin/nvim --version
NVIM v0.10.4
Build type: Release
LuaJIT 2.1.1713773202
Run "nvim -V1 -v" for more info
```
The setup function for ccc is

```lua
local ccc = require("ccc")
ccc.setup {
	highlighter = {
		auto_enable = true,
		max_byte = 2 * 1024 * 1024, -- 2mb
		lsp = true,
		filetypes = colorPickerFts,
	},
	pickers = {
		ccc.picker.hex,
		ccc.picker.css_rgb,
		ccc.picker.css_hsl,
		ccc.picker.ansi_escape {
			meaning1 = "bright", -- whether the 1 means bright or yellow
		},
	},
	alpha_show = "hide", -- needed when highlighter.lsp is set to true
	recognize = { output = true }, -- automatically recognize color format under cursor
	inputs = { ccc.input.hsl },
	outputs = {
		ccc.output.css_hsl,
		ccc.output.css_rgb,
		ccc.output.hex,
	},
	convert = {
		{ ccc.picker.hex, ccc.output.css_hsl },
		{ ccc.picker.css_rgb, ccc.output.css_hsl },
		{ ccc.picker.css_hsl, ccc.output.hex },
	},
	mappings = {
		["q"] = ccc.mapping.quit,
		["L"] = ccc.mapping.increase10,
		["H"] = ccc.mapping.decrease10,
	},
}
```

I can confirm that the `fix/ccc` branch works. Otter throws a warning, but there's no error and the plugin functions normally after the warning. I'll paste the warning below

![Image](https://github.com/user-attachments/assets/f55e7f01-b36b-4f66-8876-b53aca0a106e)
Does it look right?

**Comment by jmbuhr** [comment]
ah, that explains why I couldn't reproduce this. The issue disappears in nvim v0.11.

**Comment by jmbuhr** [comment]
the fix is merged now.
